node/src/accountant/mod.rs

```diff
@@ -1458,73 +1396,69 @@ pub struct ThresholdUtils {}
 
 impl ThresholdUtils {
     pub fn slope(payment_thresholds: &PaymentThresholds, for_wei_computation: bool) -> f64 {
-        if payment_thresholds.debt_threshold_gwei <= payment_thresholds.permanent_debt_allowed_gwei
-        {
-            panic!(
-                "Debt threshold ({}) cannot be smaller than permanent debt allowed ({})",
-                payment_thresholds.debt_threshold_gwei,
-                payment_thresholds.permanent_debt_allowed_gwei
-            )
-        }
-        let slope = -(payment_thresholds.debt_threshold_gwei as f64
-            - payment_thresholds.permanent_debt_allowed_gwei as f64)
+        let slope = (payment_thresholds.permanent_debt_allowed_gwei as f64
+            - payment_thresholds.debt_threshold_gwei as f64)
             / Self::convert(payment_thresholds.threshold_interval_sec) as f64;
         if for_wei_computation {
-            slope * 10_u64.pow(9) as f64
+            slope * WEIS_OF_GWEI as f64
         } else {
             slope
         }
     }
 
-    fn convert(num: u64) -> i64 {
-        checked_conversion::<u64, i64>(num)
-    }
-
-    fn compute_theoretical_interception_with_y_axis(
-        m: f64,
-        debt_threshold: i64,
-        maturity_threshold_sec: i64,
-    ) -> f64 {
-        let m = m.abs();
-        let x_interception = (debt_threshold as f64 / m) + maturity_threshold_sec as f64;
-        m * x_interception
-    }
-
-    pub fn calculate_sloped_threshold_by_time(
+    fn calculate_sloped_threshold_by_time(
```

Comment:
There's a fair amount of integer / floating-point conversion going on here. We might be able to keep this in the integer domain by using a big factor (perhaps WEIS_OF_GWEI) to jack up the slope so that it's not so small that floating-point is necessary. We might run into small rounding errors, but as long as they're small, nobody will care, since this isn't about stored debt but rather about delinquency detection.

Here's what I wrote during the review. My counsel would be to spend a small amount of time doing this on an experimental branch, and if it causes more than a little trouble, make a separate card out of it, because it is an optimization that can be applied at any time later.

```
slope: f64 = dY as f64 / dX as f64

let FACTOR = 1_000_000_000;

int_slope: i64 = (dY * FACTOR) / dX

int_y = slope * x + (b * FACTOR);

y = int_y / FACTOR
```